index.html

```diff
@@ -14,18 +14,31 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
   </head>
   <body>
-    <header>
+    <header class="header-container active">
+      <h2 class="my-logo">My logo</h2>
+      <button  type="button" id="menu-bar"><i class="fa fa-bars"></i></button>
       <nav class="nav">
-        <h2 class="my-logo">My logo</h2>
-        <img src="assets/Unionnavbar.png" alt="nav-bar" class="navbar">
         <ul class="nav-list">
-          <li class="nav-item1">Portfolio</li>
-          <li class="nav-item2">About</li>
-          <li class="nav-item3">Contact</li>
+          <li class="nav-item1"><a href="#"></a>Portfolio</li>
+          <li class="nav-item2"><a href="#"></a>About</li>
+          <li class="nav-item3"><a href="#"></a>Contact</li>
         </ul>
       </nav>
```

Comment:
- [ ] [REQUIRED] Good job so far!  The `Mobile menu` is working well, but you still need to work on some parts of it. When clicking the `hamburger` button and scrolling, the `background color` stays in place and moves to other sections of the project. Please, when you start the mobile menu and scroll, the mobile menu background picture should not disappear. Finally after clicking on `close`(`X`) button of mobile menu and other buttons in `menu` header page is showing `blur` view please fix this also 🙏🏼🙏🏼🙏🏼

![image](https://user-images.githubusercontent.com/63915024/215553451-19be1b7e-cf01-41b1-86f2-7e3df8cb7ea3.png)